### PR TITLE
[lit-next] Fixes queryAssignedNodes when querying for nodes on browsers that do …

### PR DESCRIPTION
**Pull request by sorvell**
…not support Element.matches

Fixes https://github.com/Polymer/lit-element/issues/1088

### CODE REVIEW

File: packages/updating-element/src/decorators/queryAssignedNodes.ts [hunk 1]
```diff
@@ -80,6 +80,7 @@ export function queryAssignedNodes(
           nodes = nodes.filter(
             (node) =>
               node.nodeType === Node.ELEMENT_NODE &&
+              // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (justinfagnani) [inline packages/updating-element/src/decorators/queryAssignedNodes.ts:83]: Free bytes: remove the whole check!

https://developer.mozilla.org/en-US/docs/Web/API/Element/matches#Browser_compatibility

**Review by justinfagnani — CHANGES_REQUESTED** [review]

File: packages/updating-element/src/decorators/queryAssignedNodes.ts (hunk 1 above)
Comment (sorvell) [reply packages/updating-element/src/decorators/queryAssignedNodes.ts:83]: IE11 uses the prefix.

Comment (justinfagnani) [reply packages/updating-element/src/decorators/queryAssignedNodes.ts:83]: argh... can we push this to the polyfills?

Comment (sorvell) [reply packages/updating-element/src/decorators/queryAssignedNodes.ts:83]: Definitely: https://github.com/webcomponents/polyfills/issues/397

**Review by kevinpschaaf — APPROVED** [review]
LGTM

**Review by justinfagnani — APPROVED** [review]

**Pull request merged**